### PR TITLE
feat(locale): add German translation 

### PR DESCRIPTION
**Pull request by vsteffan**
Added a file de.yaml with german translations.

### CODE REVIEW

File: locales/de.yaml [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+    description: See and edit your account's details for the Bodega Map
+  register:
+    title: Register
+    description: Create an account and enjoy the user-only features of the Bodega Map
+  verify:
+    title: E-mail verification
+    description: E-mail verification page for Bodega Map accounts
+  recovery:
+    title: Password recovery
+    description: Password recovery page for Bodega Map accounts
```
Comment (tmlmt) [inline locales/de.yaml:271]: Forgot to point out that this section also needs translation. Would you mind translating these last few strings? Thanks !

**Comment by tmlmt** [comment]
Amazing work, @vsteffan ! 🔥
I just realized that a few extra strings need translation. See comment. I'll review afterwards :)

**Comment by vsteffan** [comment]
Wesh. I translated the meta data also now - does it work?

**Comment by tmlmt** [comment]
Hmmm. Unfortunately, I don't see the meta translations. Your latest commit has just added a line break ... 🤔

**Comment by vsteffan** [comment]
Maybe now? I pushed the remaining changes! :)

**Review by tmlmt — APPROVED** [review]
Looking good !

**Pull request merged**